frontend/cypress/integration/common/hooks.ts

```diff
@@ -9,3 +9,41 @@ Before({tags: '@gateway-api'}, async function () {
     }
   })
 });
+
+// Will be refactored soon
```

Comment:
Please roll back the file to its original state. It's not a good practice to commit TODOs and unused code.